src/tape/ensemble.py

```diff
@@ -1142,6 +1160,65 @@ def s2n_inter_quartile_range(flux, err):
                 meta=meta,
             )
 
+        # Output standardization
```

Comment:
It would be helpful for readability if you could brake this code into few functions

Reply:
I've broken all the formatting into a separate function, let me know if you think this should go further.

Reply:
Thank you, `batch` is much more readable now! That new function is still a bit hard to understand, may you please add more comments/helper functions there?

Reply:
I've paired down the logic and added additional comments in the latest commit. Hopefully that helps!